WalletWasabi/TorSocks5/TorProcessManager.cs

```diff
@@ -68,47 +68,47 @@ public void Start(bool ensureRunning, string dataDir)
 						}
 
 						var fullBaseDirectory = Path.GetFullPath(AppContext.BaseDirectory);
-						if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
-						{
-							if (!fullBaseDirectory.StartsWith('/'))
-							{
-								fullBaseDirectory.Insert(0, "/");
-							}
-						}
-
```

Comment:
Regarding the rest of the code in this file from here I'm really not sure which one is the more readable code.

Reply:
I find `If (condition) {do something} else {do something else}` easier to read than `If (!condition) {do something else} else {do something}` but this is just my opinion.